lib/core/preferences.dart

```diff
@@ -0,0 +1 @@
+ const String isLoggedIn="isLoggedIn";
```

Comment:
Please make a class of this and declare this in string file not in in core under utils make a class of all string and put there and Please read about the why to make class .